src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -65,6 +66,12 @@ public EventTypeController(EventTypeRepository eventTypeRepository, TopicReposit
         }
 
         try {
+            // can happen if EventType already exists or if topic deletion
+            // was not yet processed after EventType deletion
+            if (topicRepository.topicExists(eventType.getName())) {
```

Comment:
good point. Implemented it with this commit https://github.com/zalando/nakadi/commit/7cb7a5bf3693e1bc9db7ba4666c400996da2274c